mciem-member-projects.sh

```diff
@@ -14,84 +34,82 @@ for GCP_COLLECTION_PROJECT_ID in $(echo $GCP_COLLECTION_PROJECT_IDS | tr "," "\n
       --member="serviceAccount:${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com" \
-      --role="roles/iam.roleAdmin"
+      --role="organizations/${GCP_COLLECTION_ORG_ID}/roles/CloudKnoxIAMOrgAdmin"
   fi  
 
 done
 
-
-echo Adding permissions to folders $GCP_COLLECTION_ORG_IDS and set controller flag to $MCIEM_GCP_ENABLE_CONTROLLER
-for GCP_COLLECTION_ORG_ID in $(echo $GCP_COLLECTION_ORG_IDS | tr "," "\n"); do
-  echo Add IAM roles/iam.securityReviewer policy binding for iam.securityReviewer to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_ORG_ID}.iam.gserviceaccount.com
-  gcloud resource-manager folders add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
+echo Adding permissions to org $GCP_COLLECTION_ORG_ID and set controller flag to $MCIEM_GCP_ENABLE_CONTROLLER
+if  [ ! -z "$GCP_COLLECTION_ORG_ID" ]; then 
+  echo Adding IAM policy binding iam.securityReviewer to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_ORG_ID}.iam.gserviceaccount.com
+  gcloud organizations add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
       --member="serviceAccount:${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com" \
-      --role="roles/iam.securityAdmin"
+      --role="roles/iam.securityReviewer"
 
-  echo Add IAM roles/iam.roleAdmin policy binding for iam.roleAdmin to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com 
-  gcloud resource-manager folders add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
+  echo Adding Viewer Role to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com 
+  gcloud organizations add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
     --member="serviceAccount:${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com" \
-    --role="roles/iam.roleAdmin"
+    --role="roles/viewer"
 
   if [ $MCIEM_GCP_ENABLE_CONTROLLER = 'y' ];
   then
     echo Enabling controller for folder $GCP_COLLECTION_ORG_ID
-    echo Add IAM roles/iam.securityAdmin policy binding for iam.securityAdmin to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com 
-    gcloud resource-manager folders add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
+    echo Adding IAM policy binding iam.securityAdmin to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com 
+    gcloud organizations add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
       --member="serviceAccount:${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com" \
       --role="roles/iam.securityAdmin"
     
-    echo Add IAM roles/iam.roleAdmin policy binding for iam.roleAdmin to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com 
-    gcloud resource-manager folders add-iam-policy-binding ${GCP_COLLECTION_ORG_ID} \
+    echo Adding IAM policy binding iam.roleAdmin to ${GCP_OIDC_SERVICE_ACCOUNT_NAME}@${GCP_OIDC_PROJECT_ID}.iam.gserviceaccount.com 
```

Comment:
can we change the echo: say we are adding CloudKnoxIAMOrgAdmin

Reply:
yup